test/Traversals/depthfirst.jl

```diff
@@ -3,7 +3,14 @@
     g5 = SimpleDiGraph(4)
     add_edge!(g5, 1, 2); add_edge!(g5, 2, 3); add_edge!(g5, 1, 3); add_edge!(g5, 3, 4)
     gx = cycle_digraph(3)
-  
+    gt = SimpleDiGraph(3)
```

Comment:
Comment before this line explaining we're testing cycles.

Also - define these graphs immediately before the `@testset` where they're being used, not all up front like this.